### PR TITLE
[#214] remove docker from edi

### PR DESCRIPTION
**Pull request by alexander-semenets**
EDI uses docker utility to pull image and get meta from labels. That was replaced with python call to docker registry.

### CODE REVIEW

File: legion/tests/test_k8s.py [hunk 1]
```diff
@@ -1,124 +1,145 @@
...
-
-        self.assertEqual(k8s_name, 'model-{}-{}'.format(model_id, expected_model_version))
-
-        self.assertIsInstance(compatible_labels, dict, 'labels is not a dict')
-        self.assertTrue(compatible_labels, 'empty labels')
-        for key, value in compatible_labels.items():
-            self.assertIsInstance(key, str, 'labels key is not a string')
-            self.assertIsInstance(value, str, 'labels key is not a string')
-
-    def test_container_id_detection_kubernetes(self):
```
[the remark this answers is not in the document]
Comment (alexander-semenets) [reply legion/tests/test_k8s.py]: done

**Comment by alexander-semenets** [comment]
CI ok to test

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
ok to test

**Comment by akharlamov** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Comment by alexander-semenets** [comment]
test this please

**Pull request merged**